### PR TITLE
[CLD-2216]Add gp3 storageclass

### PR DESCRIPTION
**Pull request by andrleite**
#### Summary
Add gp3 StorageClass

#### Ticket Link

  Fixes https://mattermost.atlassian.net/browse/CLD-2216

#### Release Note

```release-note
Add StorageClass for type gp3
```

### CODE REVIEW

**Review by d-wierdsma — APPROVED** [review]

File: aws/cluster/storage_class.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    annotations = {
+      "storageclass.kubernetes.io/is-default-class" = var.is_gp3_default
+    }
+  }
+  storage_provisioner    = "ebs.csi.aws.com"
+  reclaim_policy         = "Delete"
+  volume_binding_mode    = "WaitForFirstConsumer"
+  allow_volume_expansion = true
+  parameters = {
+    type = "gp3"
```
Comment (stafot) [inline aws/cluster/storage_class.tf:13]: Please make this variable with default value gp3

**Review by stafot — APPROVED** [review]

File: aws/cluster/storage_class.tf (hunk 1 above)
Comment (andrleite) [reply aws/cluster/storage_class.tf]: done!

**Review by d-wierdsma — APPROVED** [review]

**Review by stafot — APPROVED** [review]

**Comment by stafot** [comment]
@andrleite is there any blocker here? Let's merge it and proceed.

**Comment by andrleite** [comment]
> 

No blockers, we can merge it. Please approve again I had to fix conflicts.

**Review by stafot — APPROVED** [review]

**Pull request merged**